bin/console-conf-wrapper

```diff
@@ -84,12 +84,26 @@ if [ "$(snap managed)" = "true" ]; then
     exit 0
 fi
 
-cat /usr/share/subiquity/console-conf-wait
+# if there is console-conf snap, exit gracefully
+if [ ! -e "/snap/console-conf" ]; then
```

Comment:
The comment and code don't seem to match, so I don't understand the intent.  Please clarify.

Reply:
Ups this is an inverted comment, it should be
`# if there is NO console-conf snap, exit gracefully`

Reply:
BTW this entire file will be likely removed as we won't be pulling subiquity to `core-base` build anymore, so this is to keep things in sync.